packages/docs/docs/docs/writing_contracts.md

```diff
@@ -249,4 +249,25 @@ Then modifying `MyContract` by swapping the order in which the base contracts ar
 contract MyContract is B, A { }
 ```
 
-Violating any of these storage layout restrictions will cause the upgraded version of the contract to have its storage values mixed up, and can lead to critical errors in your application.
\ No newline at end of file
+You also cannot add new variables to base contracts, if the child has any variables of its own. Given the following scenario:
+```js
+contract Base {
+  uint256 base1;
+}
+
+contract Child {
+  uint256 child;
+}
+```
+
+And if `Base` is modified to add an extra variable:
```

Comment:
I find the "and" at the beginning of the sentence a bit confusing when the section is read as a whole.